data/autoyast_sle15/autoyast_wicked_ppc64le.xml

```diff
@@ -63,6 +63,7 @@
             sed -i 's/splash=silent\ quiet//' /etc/default/grub
             sed -i '/GRUB_DISABLE_LINUX_UUID/s/^# //' /etc/default/grub
             grub2-mkconfig -o /boot/grub2/grub.cfg
+            systemctl enable serial-getty@hvc1
```

Comment:
Lets enable `hvc2` as well, to get offline upload working in the second console. WDYT?

Reply:
hm , the problem that it would be unconditional enable , which contradict with the fact that VIRTIO_CONSOLE_NUM is actually conditional variable . Let me try something different in this regard ...

Reply:
please check what I did in `wickedbase` is it ok like this ?

Reply:
ok